spec/advanced_search_spec.rb

```diff
@@ -130,16 +130,16 @@
         expect(@sub_no_phrase).to have_the_same_number_of_results_as(solr_resp_doc_ids_only(subject_search_args('home schooling')))
       end
       it 'subject as a phrase' do
-        expect(@sub_phrase.size).to be >= 500
-        expect(@sub_phrase.size).to be <= 605
+        expect(@sub_phrase.size).to be >= 550
+        expect(@sub_phrase.size).to be <= 615
         expect(@sub_phrase).to have_fewer_results_than(solr_resp_doc_ids_only(subject_search_args("home schooling")))
         expect(@sub_phrase).to have_fewer_results_than @sub_no_phrase
       end
       it 'keyword' do
         resp = solr_resp_doc_ids_only({ 'q' => 'Socialization' }.merge(solr_args))
-        # resp.should have_the_same_number_of_results_as(solr_resp_ids_from_query('Socialization'))
-        expect(resp.size).to be >= 475_000
-        expect(resp.size).to be <= 525_000
+        expect(resp).to have_fewer_results_than(solr_resp_ids_from_query('Socialization'))
```

Comment:
I no longer remember why this is true -- that keyword in advanced search would have fewer results than plain everything search.  But I trust you.

Reply:
The solr query sent for advanced search uses defType=lucene, hence fewer results.

Reply:
ah yes - should there be a comment to that effect?